util/travis_push.sh

```diff
@@ -5,13 +5,15 @@ TRAVIS_PULL_REQUEST="${TRAVIS_PULL_REQUEST:false}"
 TRAVIS_COMMIT_MESSAGE="${TRAVIS_COMMIT_MESSAGE:-none}"
 TRAVIS_COMMIT_RANGE="${TRAVIS_COMMIT_RANGE:-HEAD~1..HEAD}"
 
-git config --global user.name "QMK Bot"
-git config --global user.email "hello@qmk.fm"
+if [[ "$TRAVIS_BRANCH" == "master" && "$TRAVIS_PULL_REQUEST" == "false" ]] ; then
+	git config --global user.name "QMK Bot"
```

Comment:
Is setting it in `--global` necessary?

Reply:
@mechmerlin not sure i makes much difference on travis, as its a clean vm per build. I kept as much of the logic the same, to minimise the change as much as i could.